### PR TITLE
add initial details about theta testnet

### PR DESCRIPTION
**Pull request by uditvira**

### CODE REVIEW

**Comment by okwme** [comment]
still needs public endpoints recorded here

File: v7-theta/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+### Add to Keplr
+
+Use this [jsfiddle](https://jsfiddle.net/hba2rxd0/4/).
+
+**NOTE:** We'll be re-running these devnets regularly with fresh state. If you'd like us to add you to our genesis accounts each time, please make a PR with a cosmos address and stating that you'd like to be automatically included in future devnets.
+
+## Theta Public testnet
+
+We're running a public testnet to help validators participated in a simulated upgrade to the `v7-theta` release candidate before upgrade on the mainnet.
```
Comment (yaruwangway) [inline v7-theta/README.md:38]: `v7-theta` -> `v7-Theta`

File: v7-theta/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# v7-theta Testnets
+
+As per the Cosmos Hub [roadmap](https://github.com/cosmos/gaia/blob/main/docs/roadmap/cosmos-hub-roadmap-2.0.md) Gaia v7-theta (v7.0.x) is planned to be released in Q1 2022 and contains the following:
+- Cosmos SDK v0.45
+- Gravity DEX: Liquidity v1.4.5
+- IBC 3.0.0
+- Interchain Account Module
+
+The chain-id will remain `cosmoshub-4` during the upgrade.
```
[the remark this answers is not in the document]
Comment (uditvira) [reply v7-theta/README.md:9]: Yes I believe the chain-id stays the same in an upgrade using the software upgrade proposal type of the gov module

**Comment by uditvira** [comment]
Posted in slack but adding for visibility. When I try to do a software upgrade using the theta-prepare branch, I'm currently getting the following errors:

```
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: 5:19PM INF start to run module migrations...
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: 5:19PM INF minted coins from module account amount=20421409uatom from=mint module=x/bank
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: 5:19PM INF executed block height=9035850 module=consensus num_invalid_txs=0 num_valid_txs=0
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: panic: cannot delete latest saved version (1)
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: goroutine 1 [running]:
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: github.com/cosmos/cosmos-sdk/store/rootmulti.(*Store).pruneStores(0xc00012a580)
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]:         github.com/cosmos/cosmos-sdk@v0.45.1/store/rootmulti/store.go:430 +0x1b6
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: github.com/cosmos/cosmos-sdk/store/rootmulti.(*Store).Commit(0xc00012a580)
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]:         github.com/cosmos/cosmos-sdk@v0.45.1/store/rootmulti/store.go:404 +0x175
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: github.com/cosmos/cosmos-sdk/baseapp.(*BaseApp).Commit(0xc00130a340)
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]:         github.com/cosmos/cosmos-sdk@v0.45.1/baseapp/abci.go:308 +0x231
Mar 04 17:19:29 local-testnet-test cosmovisor[25898]: github.com/tendermint/tendermint/abci/client.(*localClient).CommitSync(0xc0010a2720)
```

**Comment by uditvira** [comment]
> `panic: cannot delete latest saved version (1)` 

Ok regarding my last message, I've isolated this error to pruning. If I set pruning = "nothing" in my app.toml file, I no longer see any failures.

Anyone have thoughts about why this could be happening @yaruwangway @okwme?

**Comment by dasanchez** [comment]
I have some more data after attempting the upgrade on two local testnets:
1. Using a modified genesis file ([from the Hypha testnet repo](https://github.com/hyphacoop/testnets/tree/add-theta-testnet/v7-theta/local-testnet))
The upgrade went through OK at block height `9035025`, but when it reached `9035030` I got this error:
`Mar 07 20:00:07 dante-local-theta-upgrade cosmovisor[39794]: 8:00PM ERR CONSENSUS FAILURE!!! err="cannot delete latest saved version (6)" module=consensus stack="goroutine 165 [running]:\nruntime/debug.Stack()\n\truntime/debug/stack.go:24 +0x65\ngithub.com/tendermint/tendermint/consensus`
The full log can be found [here](https://hackmd.io/@dasanchez/Skt4ZWS-5).

2. Using a "fresh" genesis file
I only modified the voting parameters in `genesis.json` to make the upgrade process easier. 
The upgrade went through OK at block height 50 and has gone past 750 now.
The steps I followed can be found [here](https://hackmd.io/@dasanchez/HJz1EelW5).

So far, it _seems_ like the pruning fails on an upgrade when we have some non-zero block height.

**Comment by yaruwangway** [comment]
> > `panic: cannot delete latest saved version (1)`
> 
> Ok regarding my last message, I've isolated this error to pruning. If I set pruning = "nothing" in my app.toml file, I no longer see any failures.
> 
> Testing with or without cosmovisor makes no difference.
> 
> Anyone have thoughts about why this could be happening @yaruwangway @okwme?

hi, theta upgrade only has one new module, ica, so i removed ica, then there is no panic at and after upgrade.

**Comment by yaruwangway** [comment]
i reported in ibc repo https://github.com/cosmos/ibc-go/issues/1088

**Comment by dasanchez** [comment]
I tested the upgrade on a local testnet again:
- using [this commit](https://github.com/cosmos/gaia/pull/1219/commits/aa76c2577c5ebc7f7b0eea5a643d7de1ad3f0e93) from `theta-prepare`
- modified genesis file (block height starts at 9034670)

The upgrade succeeded, and there is no panic 200 blocks later.
I posted the log [here](https://hackmd.io/@dasanchez/B1kEnmI-q).

**Pull request merged**